release-notes/2024-12/README.md

```diff
@@ -0,0 +1,198 @@
+# SystemLink Enterprise 2024-12 Release Notes
+
+The 2024-12 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Behavior change or new feature description
+
+- Updated **Automated Agent** built-in role to include privileges to **Execute test actions**.
+
+## Helm Chart Breaking Changes
+
+- `dynamicformfields:0.1.310`
+    - The DynamicFormFields (DFF) Service was added to the top-level chart.
```

Comment:
In the User Manual I have been using the term "Dynamic Form Fields" with spaces. Perhaps we should be consistent.